composer.json

```diff
@@ -13,7 +13,7 @@
 		"issues": "https://github.com/friendica/friendica/issues"
 	},
 	"require": {
-		"php": ">=7.0",
```

Comment:
What feature made you bump this?

Reply:
https://packagist.org/packages/phpunit/phpunit#7.5.0 <-- this one :)

Reply:
Why do we need to upgrade to this version?

Reply:
https://phpunit.de/supported-versions.html

Because eveb 7.5.0 is EOL and doesn't "officially" supports PHP 8.0 but at least supports PHP 7.1 (and worked with our tests)
The next step would be PHPUnit 8, which starts with PHP 7.2 but would officially support PHP 8.0

Reply:
We can have dev dependencies that are based on the executable PHP version and normal dependencies that are locked in the lowest version necessary (currently 7.0). The CI just needs to run `composer update --ignore-platform-reqs` to get the latest PHPUnit while the rest of the libraries will be bound to their own version limits.

Reply:
Even better, the CI can run `composer install --no-dev` and then `composer update phpunit/phpunit --ignore-platform-reqs` to limit the number of package updates.

Reply:
So that means that we support more versions than we're currently are able to test? Sounds a little bit risky and could lead to unseen problems.
=> But I'll do so :-)

Reply:
ok at least it's the same like we're already doing because we start with the CI with 7.2 .. so nevermind, you're right! :)

Reply:
There are two levels of support: the technical level where the codebase *can* run on PHP 7.0+. And there's the official level of support at the lowest actively supported PHP version, currently it's 7.3+. This means that we reserve the right to discard any support requests on PHP versions 7.0-7.2 even though Friendica can work on them.

So we should drop the tests on version 7.2 and open them for version 8.0.

Reply:
Puh .. I'll have to investigate a little bit more for this setup.

Because installing PHPunit without limitation of the platform will use all the "new" php 8.0 things for PHPunit as well -> This breaks our testcases, because they're inheriting from these PHP classes and now are incompatible (f.e. they're missing the new return types).

Reply:
reverted platform minimum requirements => solved, see https://github.com/friendica/friendica/pull/10112#discussion_r605983300 for followup :D